include/circt/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.h

```diff
@@ -0,0 +1,32 @@
+//===- HandshakeToFIRRTL.h --------------------------------------*- C++ -*-===//
```

Comment:
This seems likely to be the only header in this directory.  You might consider moving it up a level to: `include/circt/Conversion/HandshakeToFIRRTL.h`

Reply:
Make sense to me. StandardToHandshake.h is in a similar situation, I can move both of them to `include/circt/Conversion` in another patch.